containers/bash-scheduled-job/app/server.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+### Create the response FIFO
+rm -f response
+mkfifo response
+
+function handle_scheduled_job() {
+  RESPONSE=$(echo -e 'HTTP/1.1 200 OK\r\n');
+  bash script.sh;
+}
+
+function handle_handshake() {
+  RESPONSE=$(echo -e 'HTTP/1.1 200 OK\r\n');
+}
+
+function handleRequest() {
+
+  ## Read request, parse each line and breaks until empty line
+  while read line; do
+    echo "$line"
+    trline=$(echo "$line" | tr -d '[\r\n]')
+
+    [ -z "$trline" ] && break
+
+    HEADLINE_REGEX='(.*?)\s(.*?)\sHTTP.*?'
+    [[ "$trline" =~ $HEADLINE_REGEX ]] &&
+      REQUEST=$(echo "$trline" | sed -E "s/$HEADLINE_REGEX/\1 \2/")
+  done
+
+  ## Route to the response handler based on the REQUEST match
```

Comment:
In general it's not good to include variable names in comments (as someone renaming the variable usually forgets to update the comment).

Instead something like 

```suggestion
  # Route to the request based on the HTTP method
```